ion/textreader.go

```diff
@@ -237,7 +237,11 @@ func (t *textReader) nextBeforeTypeAnnotations() (bool, error) {
 				if err := t.verifyUnquotedSymbol(val, "annotation"); err != nil {
 					return false, err
 				}
+			} else if tok == tokenSymbolOperator {
+				return false, &SyntaxError{
+					"Annotations that include a '" + val + "' must be enclosed in quotes.", t.tok.Pos() - 1}
```

Comment:
@fernomac just [informed me on another PR](https://github.com/amzn/ion-go/pull/130#discussion_r469717690) that:

> per https://github.com/golang/go/wiki/CodeReviewComments#error-strings error strings in go should not start with a capitalized word or end in punctuation

Sorry, I know this was my suggested text. 🤦 Rather than fixing this here, I think we should make a cleanup issue to fix this project-wide before a 1.0 release.

Reply:
Opened #131 based on this comment.